.github/workflows/linux_arm64.yml

```diff
@@ -5,7 +5,7 @@ on:
   merge_group:
   push:
     branches:
-      # - master
+      - master
```

Comment:
I supposed we can take artifacts from merge queue

Reply:
The logic for that is TBD. For now I think we can handle 2 extra builds in post-commits